### PR TITLE
Chore/likes

### PR DESCRIPTION
**Pull request by hi-there-insahae**
## What is this PR do?
- `Introduction`: Add Album, Photo, Comment views
- `Notion Ticket`: [https://www.notion.so/osds-/BE-archives-82a134a6bff3418e8dc57742430423c8]()
- `Figma`: [figma link]()

## Changes
- [ ] : Add read_edit_delete_album view
- [ ] : Add delete_photo view
- [ ] : Add edit_delete_comment view
- [ ] : Refacter Like model -> Add constrains between user and photo
- [ ] : Ffix USE_TZ=False

### CODE REVIEW

**Comment by jinu-u-kim** [comment]
wow~ amazing bro you are our superstar fantastic baby

**Comment by na-yk** [comment]
좋아요 구현에 들어가시다니 좋아요~😀❤

**Comment by euije** [comment]
LGTM (Looks Good To Me) 👍

File: archives/views.py [hunk 1]
```diff
@@ -143,12 +145,49 @@ def read_edit_delete_album(request, family_id, album_id):
         )
 
     elif request.method == "PUT":
+        body = json.loads(request.body.decode('utf8'))
```
Comment (euije) [inline archives/views.py:149]: #2 Issue 참조 부탁드립니다!

File: OSDS/settings.py [hunk 2]
```diff
@@ -142,7 +142,7 @@ def get_secret(setting, secrets=secrets):
 
 USE_I18N = True
 
-USE_TZ = True
+USE_TZ = False
```
Comment (euije) [inline OSDS/settings.py:145]: ### Commit Convention 
동사를 Fix에서 Chore로 대체해주세요

**Pull request merged**